JASP-Desktop/components/JASP/Widgets/AboutWindow.qml

```diff
@@ -0,0 +1,226 @@
+import QtQuick 2.11
+import QtQuick.Window 2.11
+import QtQuick.Controls 2.5
+import QtQuick.Controls.Styles 1.4
+import QtWebEngine 1.7
+import JASP.Widgets 1.0
+import JASP.Theme 1.0
+
+Window
+{
+	id: aboutWindow
```

Comment:
The bar at the top of the About window should say "About JASP" rather than "JASP About".